src/dlgPackageExporter.cpp

```diff
@@ -281,12 +287,27 @@ bool dlgPackageExporter::eventFilter(QObject* obj, QEvent* evt)
         return true;
     }
 
+    //Focus handling returns false so that underlying class functions still work and the cursor is visible for example
+
+    //description focus handling
+    if (obj == input->Description) {
+        if (evt->type() == QEvent::FocusIn) {
+            input->Description->setPlainText(mPlainDescription);
+            return false;
+        }
+
+        if (evt->type() == QEvent::FocusOut) {
+            mPlainDescription = input->Description->toPlainText();
+            input->Description->setMarkdown(mPlainDescription);
```

Comment:
It's OK that Qt 5.11 doesn't support markdown - all official releases are on a more modern Qt that does. For the older Qt you can display markdown as-is, it's a compréhensible format.